odoo/addons/base/migrations/13.0.1.3/pre-migration.py

```diff
@@ -462,15 +462,22 @@
     ('fil', 'fil_PH'),
 ]
 
+column_renames = {
+    'res_lang': [
+        ('week_start', 'week_start_tmp'),
```

Comment:
Please rename to `None`. It will be renamed to the result of openupgrade.get_legacy_name('week_start') (and you can use that method to retrieve the renamed column name when you process its values in the post script).

Reply:
Done